spec/unit/lib/cloud_controller/diego/app_recipe_builder_spec.rb

```diff
@@ -216,7 +221,8 @@ module Diego
               global_environment_variables: env_vars,
               privileged?:                  false,
               ports:                        lrp_builder_ports,
-              port_environment_variables:   expected_action_environment_variables,
+              port_environment_variables:   expected_action_port_variables,
```

Comment:
@thausler786 @aashah Single comment to respond to all of yours :).  Makes sense, I'll modify as requested.